ansible/dom0-bootstrap.yml

```diff
@@ -54,3 +54,9 @@
       tags: on-rename
     - role: node_exporter
       tags: node_exporter
+
+- hosts: gh
+  gather_facts: false
+  roles:
+    - role: gh
+      tags: gh
```

Comment:
I'd suggest to move those bits right after `dom0-defaults`. That order feels a bit more logical.
If there is a reason for this order, I'd suggest to add a comment.

Reply:
There is no reason to have it her. I will move it up.